airflow/models/__init__.py

```diff
@@ -3249,6 +3259,17 @@ def __exit__(self, _type, _value, _tb):
 
     # /Context Manager ----------------------------------------------
 
+    @property
+    def default_view(self):
```

Comment:
Good one, added a test

Reply:
This behavior how differs from the `get_default_view` method. Maybe we can remove the `get_default_view` method completely: https://stackoverflow.com/questions/49134141/how-do-i-reference-an-object-property-in-a-method-call-in-jinja2

This would reduce the number of methods on the DAG object. Besides that, I believe that `dag.default_view` should return the same as `dag.get_default_view()` since the names are so similar.

Reply:
@Fokko Agree there. I did now remove all usage of dag.default_view and switch it to dag.get_default_view()

Reply:
@Fokko: Can't use just `default_view` because in DagModel this can be None and get_default_view has a fall back to the config when not set.

Reply:
Awesome, thanks!